src/main/java/org/myrobotlab/lang/ServoLang.java

```diff
@@ -27,10 +27,18 @@ public String toPython(Servo s) {
     // save the servo map
     sb.append(name + ".map(" + s.getMin() + "," + s.getMax() + "," + s.getMinOutput() + "," + s.getMaxOutput() + ")\n");
     // if there's a controller reattach it at rest
+    // FIXME - there is the initial position vs rest - they potentially are very different
```

Comment:
re-attaching a controller should always specify the last known position from java-land as the initial attach position/angle...  only if a servo service is created from scratch, the initial position can be defaulted to it's rest position.

Reply:
I think we agree ... servos are dumb .. if InMoov want's to remember last run of a process and where all the servo positions are .. great .. let it add that functionality ... but servos are dumb
I'm good with that and will ignore the FIXME .. I'm working on Arduino + Servo now (and always WebGui) .. and will wait until I move to InMoov to look into the details of last know position